# bazel expect Git LSF resolved sources
# And that only provided by the new URL. 
# additionally we need the original (wrong mirror) url as well:
# ERROR: Analysis of target '//tensorflow/tools/pip_package:build_pip_package' failed; build aborted: no such package
# '@icu//': tf_http_archive(urls) must have redundant URLs.
# The mirror.bazel.build URL must be present and it must come first.
# Even if you don't have permission to mirror the file, please put the correctly formatted mirror URL there anyway,
# because someone will come along shortly thereafter and mirror the file.
diff -ru tensorflow-1.13.1.orig/third_party/icu/workspace.bzl tensorflow-1.13.1/third_party/icu/workspace.bzl
--- tensorflow-1.13.1.orig/third_party/icu/workspace.bzl	2019-02-25 21:37:04.000000000 +0100
+++ tensorflow-1.13.1/third_party/icu/workspace.bzl	2021-07-06 09:31:01.496253109 +0200
@@ -14,7 +14,7 @@
         sha256 = "e15ffd84606323cbad5515bf9ecdf8061cc3bf80fb883b9e6aa162e485aa9761",
         urls = [
             "https://mirror.bazel.build/github.com/unicode-org/icu/archive/release-62-1.tar.gz",
-            "https://github.com/unicode-org/icu/archive/release-62-1.tar.gz",
+            "http://mirror.tensorflow.org/github.com/unicode-org/icu/archive/release-62-1.tar.gz",
         ],
         build_file = "//third_party/icu:BUILD.bazel",
         system_build_file = "//third_party/icu:BUILD.system",
